### PR TITLE
Add sort

### PR DESCRIPTION
**Pull request by kenji-miyake**

### CODE REVIEW

**Comment by kenji-miyake** [comment]
Tested in https://github.com/tier4/autoware-spell-check-dict/pull/3

File: .github/workflows/sort.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        run: |
+          ${GITHUB_WORKSPACE}/sort.sh
+
+      - name: Commit and push
+        run: |
+          if ! git diff --exit-code --quiet; then
+            git config user.name github-actions
+            git config user.email github-actions@github.com
+            git add -u .
+            git commit -m "Update"
```
Comment (kmiya) [inline .github/workflows/sort.yml:31]:
```suggestion
            git commit -m "Sort and uniq the words"
```
Minor: just make the commit message more concrete.

File: sort.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$(readlink -e "$(dirname "${0}")")
+
+# Install jq
+if ! (command -v jq > /dev/null 2>&1); then
+    sudo apt install -y jq
```
Comment (kmiya) [inline sort.sh:7]:
```suggestion
    sudo apt-get install -y jq
```
Minor: apt-get would be better for CLI environment.

File: sort.sh [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+# Install jq
+if ! (command -v jq > /dev/null 2>&1); then
+    sudo apt-get install -y jq
+fi
+
+# Set target file
+cspell_json="$SCRIPT_DIR/cspell/.cspell.json"
+
+# Create new json
+sorted_json=$(jq --indent 4 \
```
Comment (kmiya) [inline sort.sh:14]: Question: `.editorconfig` defines `indent_size = 2`. Is the `jq --indent 4` no problem with `.editorconfig`?

Comment (kenji-miyake) [reply sort.sh:14]: `indent_size = 2` is only for `.yml`(GitHub actions).

Comment (kmiya) [reply sort.sh:14]: Oh, yes, the setting is not for `.json`. Sorry, I just misunderstood.

**Comment by kmiya** [comment]
@kenji-miyake Thank you for the useful script!
BTW, Can I rename the `sort.sh` to `sort_words.sh` to clarify the intended use of the script?

**Comment by kenji-miyake** [comment]
@kmiya Yes, of course! I'll do it.

**Comment by kenji-miyake** [comment]
But actually, it sorts not only `words` but also `ignorePath`. `sort_json.sh` is more appropriate?

**Comment by kenji-miyake** [comment]
or `sort_dict`, `sort_json_dict`, `sort_cspell_json`?

**Comment by kenji-miyake** [comment]
As it's only for `.cspell.json`, I chose `sort_cspell_json.sh`. Is it okay for you?

**Comment by kmiya** [comment]
@kenji-miyake Yeah, it's perfect!

**Comment by kenji-miyake** [comment]
Retested https://github.com/tier4/autoware-spell-check-dict/pull/3/commits/094b96bcc41895a1c1a9185b1ff761ed7c926dac.

**Review by kmiya — APPROVED** [review]
Great! I'm happy to merge it.

**Pull request merged**